Pod/Classes/VOKCoreDataManager.m

```diff
@@ -204,15 +210,20 @@ - (void)addPersistentStoreToCoordinator:(NSPersistentStoreCoordinator *)persiste
 }
 
 - (NSManagedObjectContext *)managedObjectContextWithConcurrencyType:(NSManagedObjectContextConcurrencyType)concurrencyType
+                                                      parentContext:(nullable NSManagedObjectContext *)parentContext
 {
-    NSPersistentStoreCoordinator *coordinator = self.persistentStoreCoordinator;
-    
-    NSAssert(coordinator, @"PersistentStoreCoordinator does not exist. This is a big problem.");
-    if (!coordinator) {
-        return nil;
-    }
     NSManagedObjectContext *context = [[NSManagedObjectContext alloc] initWithConcurrencyType:concurrencyType];
-    context.persistentStoreCoordinator = coordinator;
+    if (parentContext) {
+        context.parentContext = parentContext;
```

Comment:
There's already an exception thrown if that happens so I don't think it's necessary.